docs/general/installation/macos.md

```diff
@@ -1,82 +1,101 @@
-1. Open Terminal and type `cd` followed with a space then drag the jellyfin folder into the terminal.
-1. Type `./jellyfin` to run jellyfin.
-1. Open your browser at [http://localhost:8096](http://localhost:8096)
+2. Stop the currently running server either via the dashboard or using `CTRL+C` in the terminal window.
+3. Extract the latest version into Applications
+4. Open Terminal and type `cd` followed with a space then drag the jellyfin folder into the terminal.
+5. Type `xattr -rd com.apple.quarantine .` to remove the quarantine flag.
+6. Type `codesign -fs - --deep jellyfin` to create an ad-hoc signature for the server.
+7. Type `./jellyfin` to run jellyfin.
+8. Open your browser at [http://localhost:8096](http://localhost:8096)
 
-**Uninstalling the Portable Version**
+### Uninstalling the Portable Version
 
 1. Stop the currently running server either via the dashboard or using `CTRL+C` in the terminal window.
-1. Move `/Application/jellyfin-version` folder to the Trash. Replace version with the actual version number you are trying to delete.
+2. Move `/Application/jellyfin-version` folder to the Trash. Replace version with the actual version number you are trying to delete.
+3. Delete the folder `~/.config/jellyfin/`
+4. Delete the folder `~/.local/share/jellyfin/`
 
-**Using FFmpeg with the Portable Version**
+### Using FFmpeg with the Portable Version
 
-The portable version doesn't come with FFmpeg by default, so to install FFmpeg you have three options.
+The portable version doesn't come with FFmpeg by default, so to install FFmpeg you have following options:
 
+- download the jellyfin-ffmpeg from [jellyfin-repo](https://repo.jellyfin.org/?path=/ffmpeg/macos) (recommended)
 - use the package manager homebrew by typing `brew install ffmpeg` into your Terminal ([here's how to install homebrew if you don't have it already](https://treehouse.github.io/installation-guides/mac/homebrew)
-- download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information), or
+- download the most recent static build from [this link](https://evermeet.cx/ffmpeg/get/zip) (compiled by a third party see [this page](https://evermeet.cx/ffmpeg/) for options and information) (Apple Silicon builds are not available from this source)
 - compile from source available from the official [website](https://ffmpeg.org/download.html)
 
-More detailed download options, documentation, and signatures can be found.
+Once downloaded, you need to do remove the quarantine flag for the `ffmpeg` and `ffprobe`.
 
-If using static build, extract it to the `/Applications/` folder.
+Ensure that both `ffmpeg` and `ffprobe` are located at the same path, then execute the following command:
 
-Navigate to the Playback tab in the Dashboard and set the path to FFmpeg under FFmpeg Path.
+```shell
+cd /path/to/ffmpeg/folder
```

Comment:
Tell the user to open a terminal in the path by right clicking and choose `open folder in terminal`?

Reply:
Nvm that is from automator script I forgot about that